package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todolist-v2",
+  "version": "1.0.0",
+  "description": "> Build a To Do List JavaScript application",
+  "private": true,
+  "scripts": {
```

Comment:
- [ ] The _script option_ of your _package.json_ file is **not properly configured** and hence your application does not launch in a browser when the command `npm start` is run. Kindly append this line below to the _script options:_ 
`"start": "webpack serve --open"`, in your _packge.json_ file.

**Your script option in package.json file:**
![image](https://user-images.githubusercontent.com/12745474/139962814-5f5e11c8-7293-4d95-aae5-9d155dff248d.png)